Marcio/registrar revamp (#2052)
cloudflare/cloudflare-docs: products/registrar/src/content/account-options/domain-management.md

---
order:
pcx-content: reference
---

# Domain management

You can manage your domain with Cloudflare Registrar in **Overview** on the dashboard.

## Domain status

> When your domain is registered with Cloudflare, you can review your domain status in **Overview** > **Manage Domain** in the right-hand menu. Scroll down to **Manage domain** > **Registration** tab. Here, you can review the current expiration date and Auto-Renew status for your domain.

Reviewer: ```suggestion
When your domain is registered with Cloudflare, you can review your domain status in **Overview** > **Manage Domain** in the right-hand menu. Scroll down to **Manage domain** > **Registration**. Here, you can review the current expiration date and Auto-Renew status for your domain.
```

## Editing WHOIS records

> Cloudflare redacts WHOIS information from your domain by default. However, we do store the authentic WHOIS record for your domain. You may edit the WHOIS contact data for any domain. To do that: 

Reviewer: ```suggestion
Cloudflare redacts WHOIS information from your domain by default. However, Cloudflare does store the authentic WHOIS record for your domain. You may edit the WHOIS contact data for any domain. To do that: 
```

1. Make sure you are [logged in to the Cloudflare dashboard](https://dash.cloudflare.com/login) and have selected your account / domain.
1. From the Overview app, scroll down to **Manage domain** > **Contacts**,
1. Click **Edit** in any of the contacts you previously set up. This allows you to update the contact information for the selected domain only. It will not update the contact information for other domains within the account.

## Default Contact Data

> The first time you transfer or register a new domain, a Default Contact is created. That contact information may be used for future transfers and registrations. The contact data may be updated at any time in the dashboard. Updating the Default Contact data **will not** update the contact information for any domains in the account. This Default Contact data is only used to prepopulate contact information for new registrations and transfers.

Author: I know. This is something that currently is in all caps in the dev docs, but I felt that all caps wasn't good either.

## Automatic renewal of domain

> Cloudflare Registrar enrolls your domain to auto-renew by default. Unlike other registrars, your domain will only renew at the list price set by the registry. When a domain has the auto-renew setting turned on, Cloudflare will attempt to automatically renew the domain prior to expiration. 

Author: I think there's somewhere else where we explain this. Let me check and I'll probably link to it

> There is no guarantee that the renewal will succeed. Renewals may fail for various reasons, including billing failures and registry downtime. While we will make several attempts to renew, we strongly recommend you frequently review your account to ensure your domains have been renewed.

Reviewer: ```suggestion
There is no guarantee that the renewal will succeed. Renewals may fail for various reasons, including billing failures and registry downtime. While Cloudflare will make several attempts to renew, Cloudflare strongly recommends you frequently review your account to ensure your domains have been renewed.
```
